src/viam/ur/module/ur_arm.cpp

```diff
@@ -755,9 +755,11 @@ void URArm::move_(std::vector<Eigen::VectorXd> waypoints, std::chrono::milliseco
         of.close();
         VIAM_SDK_LOG(info) << "move: end unix_time " << unix_time.count();
 
-        if (current_state_->trajectory_status.load() == TrajectoryStatus::k_cancelled) {
-            throw std::runtime_error("arm's current trajectory cancelled by code");
-        }
+        // TODO: renable this once zero velocity errors are resolved
```

Comment:
nit: Usually I put the ticket number in the todo like `TODO(RSDK-11063)`. Meanwhile, please also add a note in RSDK-11063 containing a reminder to re-enable this logic.